src/autoscaler/configutil/cf.go

```diff
@@ -193,7 +225,7 @@ func (vc *VCAPConfiguration) addPostgresConnectionParams(service *cfenv.Service,
 
 func (vc *VCAPConfiguration) addConnectionParams(service *cfenv.Service, dbName string, parameters url.Values) error {
 	// if service.Tags contains "postgres" then add the connection parameters
-	if contains(service.Tags, "mysql") {
+	if slices.Contains(service.Tags, "mysql") {
 		return nil
 		// TODO: add support for MySQL TLS enabled connections
```

Comment:
TODO ??

Reply:
I would leave this TODO as a placeholder in case there's open source interest in MySQL support for the MTAR solution. Since MTAR is already a new deployment, it's a breaking change anyway.